src/aca_main.cpp

```diff
@@ -15,37 +15,36 @@
 #include "aca_log.h"
 #include "aca_util.h"
 #include "aca_ovs_control.h"
-#include "aca_message_consumer.h"
+#include "aca_message_pulsar_producer.h"
```

Comment:
I am curious, why are we including aca_message_pulsar_producer.h instead of aca_message_pulsar_consumer.h when aca_main.cpp only do the consumer stuff?